sfdoc/publish/views.py

```diff
@@ -138,18 +140,31 @@ def review(request, pk):
         return HttpResponseRedirect('../')
     else:
         form = PublishToProductionForm()
+
+    base_url = get_community_base_url()
+    assert base_url is not None
+
+    def article_for_view(item):
+        return {
+            'preview_url': '{}{}'.format(base_url, item.preview_url),
+            'url_name': item.url_name,
+        }
+
+    def get_articles_for_view(collection):
+        return list(map(article_for_view, collection))
```

Comment:
Consider:

    return [{
        "preview_url": f"{base_url}{article.preview_url}",
        "url_name": article.url_name,
    } for article in collection]

Reply:
I write too much javascript.

Resolved by https://github.com/SFDO-Tooling/sfdoc/pull/235/commits/9e1862c180f8cfba60784e4ef50bf2cc51457346